Jenkinsfile

```diff
@@ -1,3 +1,3 @@
-standardMavenPipeline {
-	healthApisMavenImage='vasdvp/health-apis-maven:3.6-jdk-12'
+standardMavenPipeline {
+	healthApisMavenImage='vasdvp/health-apis-maven:3.6-jdk-12'
```

Comment:
Should we add `shankins` to the slackChannels?

Reply:
Yep